playbooks/common/openshift-haproxy/config.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This playbook should probably be renamed to openshift-loadbalancer/config.yml with the role renaming.